Content.Client/Fax/UI/FaxBoundUi.cs

```diff
@@ -22,12 +30,47 @@ protected override void Open()
         _window.OpenCentered();
 
         _window.OnClose += Close;
+        _window.FileButtonPressed += OnFileButtonPressed;
         _window.CopyButtonPressed += OnCopyButtonPressed;
         _window.SendButtonPressed += OnSendButtonPressed;
         _window.RefreshButtonPressed += OnRefreshButtonPressed;
         _window.PeerSelected += OnPeerSelected;
     }
 
+    private async void OnFileButtonPressed()
+    {
+        if(_dialogIsOpen)
+            return;
+        _dialogIsOpen = true;
+        var filters = new FileDialogFilters(new FileDialogFilters.Group("txt"));
+        await using var file = await _fileDialogManager.OpenFile(filters);
+
+        if(_window == null)
+        {
+            _dialogIsOpen = false;
+            return;
+        }
+        if(_window.Disposed)
+        {
+            _dialogIsOpen = false;
+            return;
+        }
+        if(file == null)
+        {
+            _dialogIsOpen = false;
+            return;
+        }
+
+        var reader = new StreamReader(file);
+        var content = await reader.ReadToEndAsync();
+        SendMessage(new FaxFileMessage(content.Substring(0, Math.Min(content.Length, 10000)), "printed paper", _window.OfficePaper));
```

Comment:
```suggestion
        using var reader = new StreamReader(file);
        var content = await reader.ReadToEndAsync();
        SendMessage(new FaxFileMessage(content.Substring(0, Math.Min(content.Length, 10000)), "printed paper", _window.OfficePaper));
```

Move 10000 to a shared system so client and server reference the same code.